docs/_includes/links.md

```diff
@@ -1,27 +0,0 @@
-<!-- Links -->
```

Comment:
markdown-magic doesn't work with MDX. Fortunately, this "include all the links we could need" pattern really isn't necessary.